src/components/organisms/SchedulePageModal/SchedulePageModal.scss

```diff
@@ -1,72 +1,60 @@
 @import "scss/constants.scss";
```

Comment:
Use variables for width/height/etc as @mike-lvov pointed out for other files in his previous review.

Reply:
used variables: https://github.com/sparkletown/sparkle/pull/1302/commits/06620c6fb5ce10998508b5403d8b8cf907ed32d6